docs/Getting Started/Installation/index.md

```diff
@@ -24,58 +24,92 @@ The new version of the Policy Server requires a SQL database. Currently the only
 
 Once you set up a database (locally or remotely) you'll need to supply the Policy Server with some environment variables. This Policy Server uses the [dotenv module](https://www.npmjs.com/package/dotenv), meaning you can write all your environment variables in a `.env` file located in the root directory of the Policy Server. The Policy Server will load the variables at `.env`. `.env` files will not be tracked by Git.
```

Comment:
This paragraph would make more sense being in the next section, since it's all about environment variables. Maybe just move the section header up to include it.